tests/factories/transcript.py

```diff
@@ -0,0 +1,29 @@
+from factory import Sequence
+from factory.alchemy import SQLAlchemyModelFactory
+
+from via.models import Transcript
+
+
+class TranscriptFactory(SQLAlchemyModelFactory):
+    class Meta:
+        model = Transcript
+
+    video_id = Sequence(lambda n: f"video_id_{n}")
+    transcript_id = Sequence(lambda n: f"transcript_id_{n}")
```

Comment:
Faker can actually [generate random langauge codes](https://faker.readthedocs.io/en/master/providers/baseprovider.html?highlight=language_code#faker.providers.BaseProvider.language_code) but don't think our transcript IDs are going to remain language codes for long so I didn't bother